tempest/api/workloadmgr/base.py

```diff
@@ -2420,3 +2420,45 @@ def restart_wlm_api_service(self):
         ssh.close()
         return status_update
 
+    '''
+    Method returns the Instance ID of a new VM instance created
+    '''
+    def create_file_manager_vm(self, vm_cleanup=True, vm_name="", security_group_id = "default", flavor_id =CONF.compute.flavor_ref, \
```

Comment:
Why is a different method required for this ? Can the existing one be reused?

Reply:
Removed method